instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -287,6 +293,45 @@ def update_expected_user_agent(expected):
         outputs = self.get_all_output()
         self.validate_outputs(outputs, modifiers=[update_expected_user_agent])
 
+    def test_traceresponse_header(self):
+        """Test a traceresponse header is sent when a global propagator is set."""
+
+        orig = get_global_response_propagator()
+        set_global_response_propagator(TraceResponsePropagator())
+
+        app = otel_asgi.OpenTelemetryMiddleware(simple_asgi)
+        self.seed_app(app)
+        self.send_default_request()
+
+        # traceresponse header corresponds to http.response.start span
```

Comment:
This one was a bit surprising to me, I'm not so sure it makes intuitive sense at first glance. I kind of feel like the span returned in the response header should be the final `"kind": trace_api.SpanKind.SERVER` span.

What do you think?

Reply:
Yes, response headers should represent the SERVER span.

Reply:
Got it, done!